### ISSUE TITLE
Bad results

### ISSUE THREAD

**Issue by deepseek** [zhang-linrui/PSENet-Plus#1]
@zhang-linrui Thank you for your hard work,
When training PSENet+ for 10 Hours on [ICDAR2015](https://rrc.cvc.uab.es/?ch=4) from scratch, I am getting bad results. 
And the loss is not getting below 0.38 even after training for hundreds of epochs! 
![img_49](https://user-images.githubusercontent.com/49877135/69092689-fbbe2400-0a5d-11ea-8422-393815dd1aae.jpg)
![r_01](https://user-images.githubusercontent.com/49877135/69092712-0b3d6d00-0a5e-11ea-8169-3b526521f55e.jpg)
![img_3](https://user-images.githubusercontent.com/49877135/69092727-1395a800-0a5e-11ea-8672-362933038bcf.jpg)

**Comment by zhang-linrui** [comment]
Thank you for your interest. For the bad results on ICDAR2015, I guess it
is because that you didn't load checkpoint properly. I once loaded a
null/wrong checkpoint file, then I got totally random bounding boxes just
like yours. If not that case, please check the tensorboard log file. The
log file should be found under the ./log folder by default. The accuracy
should be around 0.9-0.95 at least after 10 hours training, just like the
reamd's picture shows. If not, there must be some problems when training.

deepseek <notifications@github.com> 于2019年11月19日周二 上午4:53写道：

**Comment by zhang-linrui** [comment]
I test some ICDAR images with the pretrained model:
![res_enjoy_test](https://user-images.githubusercontent.com/54568644/69111246-f4c3ff80-0ab7-11ea-9388-25bd0cda71b5.jpg)
